README.md

```diff
@@ -163,15 +166,15 @@ If a published post on openliberty.io/blog contains an error or needs updating i
 
 3. If the blog tags need correcting, update the [blogs_tags.json](./blog_tags.json) file. If you add new tags, make sure to add the blog post's slug to the beginning of the `posts` arrays (1-2 entries per line) for each tag.
 
-4. Create a PR from your branch to the `draft` branch, then (when the PR has been merged) run the [draft site build from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) to check that the changes are fine on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+4. Create a PR from your branch to the `draft` branch, then (when the PR has been merged) wait for IBM Cloud to rebuild [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/) (if you have access to [this Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ), then you can use it to track build/deploy preogress.  Check that the changes are fine on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
 
-   Make any changes in your branch then push to `draft` branch again rebuild.
+   Make any changes in your branch then push to `draft` branch again verify the changes after rebuild.
 
 5. Create a PR from your branch to `staging branch` (not from `draft` branch) and add @GraceJansen as reviewer. You can create this PR at any point because any new changes you make in your branch are automatically added to the PR.
 
-6. When approved, the editor will merge to `staging`, then run the [build for the staging site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) and check that it looks fine on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/).
+6. When approved, the editor will merge to the `staging` branch, causing IBM Cloud to automatically kick off a build of both the [blogs-staging site](https://blogs-staging-openlibertyio.mybluemix.net/blog/) and [staging site](https://staging-openlibertyio.mybluemix.net/blog/), which you can use to verify the changes look correct.
```

Comment:
suggested edit:

When the PR is approved, the editor will merge it the `staging` branch, causing IBM Cloud to automatically kick off a build of both the [blogs-staging site](https://blogs-staging-openlibertyio.mybluemix.net/blog/) and [staging site](https://staging-openlibertyio.mybluemix.net/blog/), which you can use to verify the changes look correct.